### ISSUE TITLE
Can see weapons double (the item itself, and the 3d Modell); with shader getts the screen blue 

### ISSUE THREAD

**Issue by Grumpj** [FlansMods/FlansMod#743]
![2015-09-20_09 07 10](https://cloud.githubusercontent.com/assets/14369200/9980151/e6ee47f0-5f8d-11e5-9da0-67e759f02274.png)

**Comment by Grumpj** [comment]
(only 1.8)

**Comment by jamioflan** [comment]
Yep, this is on the shader mod's end. They don't check the standard Forge callback for whether to render an item before rendering their own extra shiny layer over the top
